openmdao/vectors/vector.py

```diff
@@ -589,7 +594,7 @@ def _in_matvec_context(self):
         raise NotImplementedError('_in_matvec_context not defined for vector type %s' %
                                   type(self).__name__)
 
-    def set_complex_step_mode(self, active, keep_real=False):
+    def set_complex_step_mode(self, active):
```

Comment:
the extended description in this docstring is no longer accurate

Reply:
fixed